src/ExportButton.tsx

```diff
@@ -330,7 +343,10 @@ const getStoryPayload = async (
       html_url: `iframe.html${query}`,
       description: snippetCodeAsBase64,
       variant_id: variantID,
-      args: args.current,
+      args: {
```

Comment:
Don't mix intialArgs with the vatiantArgs it's better to just pass the args.current (we use it for complex components)
if you want add an initialArgs key on the storyvariant object (might be useful in the future)

Reply:
@Datlyfe Thank you, I only have one concern :) https://www.loom.com/share/6369e2cbf5864bbdaa383af72db1fb0f

Reply:
@federico-terzi-anima  Yeah you're right as we are not waiting now for storybook internals to update the args,
my only concern is the mixing of the initialArgs with the variant args basically we want to keep them seperate so I suppose it should not be a problem to just set the args to the variant object and add a new key `initalArgs` to to the variant object, let me know if it's that's clear

Reply:
@Datlyfe Sounds good! I'll do as you suggested, thank you :) 
I've originally added this logic because the `variant` object does not contain the properties that we don't support (eg. the `label` in the Button component), so this is potentially a breaking behavior (but I'm not sure if that causes any issue, I don't know how we're using the `args` field server side) :)

Reply:
AFAIK the args key is only used on the plugin for the complex component ( because we generate a query string there too) 
so it should be fine